app/src/main/java/com/example/android/mashup/DetailsFullscreen/DetailsViewFullscreen.kt

```diff
@@ -71,116 +47,83 @@ class DetailsViewFullscreen : Fragment() {
     ): View? {
 
         _binding = FragmentDetailsViewFullscreenBinding.inflate(inflater, container, false)
-        return binding.root
-
-    }
-
-    override fun onViewCreated(view: View, savedInstanceState: Bundle?) {
-        super.onViewCreated(view, savedInstanceState)
-
-        visible = true
+//        _binding2 = PlayerControllerBinding.inflate(inflater, container, false)
+
+        val playerView = binding.player
+//        val btnExitFull = binding2.exitFullscreenBtn
+
+        simpleExoPlayer= ExoPlayer.Builder(this.requireContext()).build()
+        playerView.player = simpleExoPlayer
+        playerView.keepScreenOn = true
+//        simpleExoPlayer.addListener(object : Player.Listener{
+//            override fun onPlayerStateChanged(playWhenReady: Boolean, playbackState: Int) {
+//                super.onPlayerStateChanged(playWhenReady, playbackState)
+//            }
+//        })
+
+        val uri = resourceToUri(this.requireContext(), R.raw.video)
+
+        val videoFile = File("E:\\Repositories\\AndroidStudioProjects-for-Uni\\mashup\\app\\src\\main\\res\\raw\\video.mp4")
```

Comment:
I don't see this used anywhere, can we remove this?

Reply:
i mean i am using a mock video for now, so when i hook this up to the proper uri this will be gone

Reply:
besides val uri is used, literally in the next line:
`

        val uri = resourceToUri(this.requireContext(), R.raw.video)

        val videoFile = File("E:\\Repositories\\AndroidStudioProjects-for-Uni\\mashup\\app\\src\\main\\res\\raw\\video.mp4")
        if (uri != null){
            val mediaItem = MediaItem.fromUri(uri)
            simpleExoPlayer.setMediaItem(mediaItem)
            simpleExoPlayer.prepare()
            simpleExoPlayer.play()
        }
        else{
            Log.i("full", "uri is null")
        }
`